support/doc/production.md

```diff
@@ -136,7 +154,7 @@ server {
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
 
     # For the video upload
-    client_max_body_size 8G;
+    client_max_body_size 2G;
```

Comment:
Why ?

Reply:
Because it was in `support/nginx/peertube`, which I consider canonical.